src/base1/patternfly-overrides.less

```diff
@@ -655,4 +655,5 @@ th > label {
   /* PF4 variables */
   --pf-global--FontFamily--sans-serif: @font-family-base;
   --pf-global--FontFamily--monospace: @font-family-monospace;
+  --pf-global--FontFamily--heading--sans-serif: @font-family-base;
```

Comment:
This doesn't _quite_ fix it. This line says to use the body fonts for the headings. Instead of RedHatText, the headings should be using RedHatDisplay.

Also, this is only a partial fix. Opting into the Red Hat font with PF4 also changes a bunch of other stuff listed @ https://www.patternfly.org/v4/documentation/overview/red-hat-font, such as:

- Changing the weights used all across PF4 widgets (in different ways in different widgets)
- Changing font sizes for headings to be better metric-wise
- Line height change for some widgets

Reply:
Fixed in https://github.com/cockpit-project/cockpit/pull/13198